hphp/runtime/vm/jit/abi-arm.cpp

```diff
@@ -89,9 +89,15 @@ const RegSet kScratchCrossTraceRegs =
 /*
  * Helper code ABI registers.
  */
-const RegSet kGPHelperRegs = rAsm | vixl::x14;
+const RegSet kGPHelperRegs = vixl::x14 | rAsm;
 const RegSet kSIMDHelperRegs = vixl::d5 | vixl::d6 | vixl::d7;
 
+/*
+ * Fast Stub ABI registers.
+ */
+const RegSet kGPFastStubRegs = vixl::x16 | rAsm;
```

Comment:
There's a comment in `abi-arm.cpp` that reads:

```
// x16 = used as ip0/tmp0 by MacroAssembler
```

How do you know that letting XLS use `x16` won't cause conflicts with whatever the assembler (vixl?) is doing with `x16`?